scripts/deployResultManagerMerkle.js

```diff
@@ -35,9 +35,9 @@ async function main() {
     const tx2 = await transparentForwarder.setForwarder(forwarder.address);
     await tx2.wait();
     console.log("Transaction hash: ", tx2.hash);
-    console.log("Sleeping for 1 minute...")
-    await sleep(60000);
-    console.log("Awake...");
+    // console.log("Sleeping for 1 minute...")
```

Comment:
@rajkharvar reduce this to 30 seconds instead of removing it, interacting with a contract right after deployment can cause issues on some chains.